### DISCUSSION TITLE
Welcome to WeakAuras-WotLK Discussions!

### DISCUSSION THREAD

**Discussion by Bunny67** [Bunny67/WeakAuras-WotLK#59] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Titousky** [comment]
Hi, I'm using the latest backported version of weakaura for 3.3.5a (on server Warmane Lordaeron) version 3.2.3 downloaded here : https://github.com/Bunny67/WeakAuras-WotLK

But I just realized that sometimes the WA icons have HUGE delay !
For example, I made a Sinister Strike icon to appears whenever it is usable for my rogue, but sometimes the icon appears up to 5 seconds after the action has really become usable... any solution?
Video of the issue : https://drive.google.com/file/d/1T0-E6bV3j6RjJt7EOND5cZ-Lf7ZFctA9/view?usp=sharing

**Reply by Bunny67** [reply]
> Hi, I'm using the latest backported version of weakaura for 3.3.5a (on server Warmane Lordaeron) version 3.2.3 downloaded here : https://github.com/Bunny67/WeakAuras-WotLK
> 
> But I just realized that sometimes the WA icons have HUGE delay ! For example, I made a Sinister Strike icon to appears whenever it is usable for my rogue, but sometimes the icon appears up to 5 seconds after the action has really become usable... any solution? Video of the issue : https://drive.google.com/file/d/1T0-E6bV3j6RjJt7EOND5cZ-Lf7ZFctA9/view?usp=sharing

Possible fix in https://github.com/Bunny67/WeakAuras-WotLK/commit/54db65937dfb8a9002e13a31f92f85abc45d749c

**Comment by beeerya** [comment]
Hi, I'm using the latest backported version of weakaura for 3.3.5a (on server WOW Sirus) version 3.2.3 downloaded here : https://github.com/Bunny67/WeakAuras-WotLK

Energy Bar is restored in jerks. Any solution?
Video of the issue : https://drive.google.com/file/d/1w49DJq1YESjAom-AewivZ2mx9juUoTwy/view?usp=sharing

**Comment by reap3rx** [comment]
Hi, I don't know if this is still being worked on or not, but I'm having an issue for the "Cast Success" combat log trigger. The trigger works fine for instant cast spells, but not spells with a cast time. For example, I can cast "Fire Blast" and it will trigger the WA, but I can not cast Fireball and trigger the WA. I'm on the Ascension free shard.

**Comment by zw24711290** [comment]
hey man love your work,I from china,so shitty english.I'm using the latest backported version of weakaura for 3.3.5a(on warmane server) ,can you add a function , Anchor point to the nameplate. thx

**Comment by Hashatan** [comment]
Hi, I love your work improving addons for WoW 3.3.5a. I have a problem: Weak Aura 4.0 is great, but it's causing a conflict with the Elvui healing addon. This addon is bugging me, so I want to have both addons.
